src/context-providers/builder/provider.tsx

```diff
@@ -4,15 +4,31 @@ import {
     TBuilderState,
     TBuilderAction,
     TBuilderContext,
-    EFormBuilderMode,
+    EBuilderMode,
 } from "./types";
 import { noop } from "lodash";
+import { TField } from "src/schemas";
+import { EFieldType } from "src/schemas/types";
 
 // =============================================================================
 // DEFAULT VALUES
 // =============================================================================
 const DEFAULT_VALUES: TBuilderState = {
-    mode: EFormBuilderMode.ADD_FIELD,
+    mode: EBuilderMode.ADD_FIELD,
+    fields: new Map<string, TField>()
```

Comment:
able to remove this since the initial state is an empty display. To test the card's look, create another story